src/p4p/asLib/__init__.py

```diff
@@ -13,6 +13,8 @@
 from .. import Value
 from ..client.thread import Context, LazyRepr, Disconnected
 
+from ..util import readnproc
```

Comment:
I'd like to keep gateway specific code out of `p4p.util`, which is for general purpose utility code.  Either keep this in `gw.py`, or move it under `asLib/`.